lib/OpenQA/WebAPI/Plugin/ObsRsync.pm

```diff
@@ -515,4 +533,24 @@ sub _for_every_batch {
     return @ret;
 }
 
+# Based on https://www.suse.com/c/multi-factor-authentication-on-suses-build-service/
+sub _bs_ssh_sign ($key, $realm, $value) {
+    die "SSH Key File not found at $key" unless (-e $key || -z $key);
+    # This needs to be a bit portable for CI testing
+    my $tmp = Mojo::File::tempfile->spew($value);
```

Comment:
Please always use a descriptive tempfile.
There should be no cryptic `/tmp/k3cQnZzpr7` files lying around (which can happen under certain circumstances where the cleanup doesn't happen).
```suggestion
    my $tmp = Mojo::File::tempfile('ssh-keyfile-XXXXX')->spew($value);
```